config.json

```diff
@@ -43,284 +43,286 @@
-    "optimizedCheckout-buttonSecondary-borderColor": "#cccccc",
-    "optimizedCheckout-buttonSecondary-backgroundColorHover": "#f5f5f5",
-    "optimizedCheckout-buttonSecondary-backgroundColorActive": "#e5e5e5",
-    "optimizedCheckout-buttonSecondary-borderColorHover": "#999999",
-    "optimizedCheckout-buttonSecondary-borderColorActive": "#757575",
-    "optimizedCheckout-link-color": "#476bef",
-    "optimizedCheckout-link-font": "Google_Montserrat_500",
-    "optimizedCheckout-discountBanner-backgroundColor": "#e5e5e5",
-    "optimizedCheckout-discountBanner-textColor": "#333333",
-    "optimizedCheckout-discountBanner-iconColor": "#333333",
-    "optimizedCheckout-orderSummary-backgroundColor": "#ffffff",
-    "optimizedCheckout-orderSummary-borderColor": "#dddddd",
-    "optimizedCheckout-step-backgroundColor": "#757575",
-    "optimizedCheckout-step-textColor": "#ffffff",
-    "optimizedCheckout-form-textColor": "#666666",
-    "optimizedCheckout-formField-borderColor": "#cccccc",
-    "optimizedCheckout-formField-textColor": "#333333",
-    "optimizedCheckout-formField-shadowColor": "transparent",
-    "optimizedCheckout-formField-placeholderColor": "#999999",
-    "optimizedCheckout-formField-backgroundColor": "#ffffff",
-    "optimizedCheckout-formField-errorColor": "#d14343",
-    "optimizedCheckout-formField-inputControlColor": "#476bef",
-    "optimizedCheckout-step-borderColor": "#dddddd",
-    "optimizedCheckout-header-borderColor": "#dddddd",
-    "optimizedCheckout-header-textColor": "#333333",
-    "optimizedCheckout-loadingToaster-backgroundColor": "#333333",
-    "optimizedCheckout-loadingToaster-textColor": "#ffffff",
-    "optimizedCheckout-link-hoverColor": "#002fe1",
-    "product_sale_badges": "none",
-    "color_badge_product_sale_badges": "#007dc6",
-    "color_text_product_sale_badges": "#ffffff",
-    "color_hover_product_sale_badges": "#000000",
-    "restrict_to_login": false,
-    "swatch_option_size": "22x22",
-    "social_icon_placement_top": false,
-    "social_icon_placement_bottom": "bottom_none",
-    "geotrust_ssl_common_name": "",
-    "geotrust_ssl_seal_size": "M",
-    "navigation_design": "simple",
-    "price_ranges": true,
-    "pdp-price-label": "",
-    "pdp-sale-price-label": "Now:",
-    "pdp-non-sale-price-label": "Was:",
-    "pdp-retail-price-label": "MSRP:",
-    "pdp-custom-fields-tab-label": "Additional Information",
-    "paymentbuttons-paypal-layout": "vertical",
-    "paymentbuttons-paypal-color": "gold",
-    "paymentbuttons-paypal-shape": "pill",
-    "paymentbuttons-paypal-size": "responsive",
-    "paymentbuttons-paypal-label": "checkout",
-    "paymentbuttons-paypal-tagline": false,
-    "paymentbuttons-paypal-fundingicons": false,
-    "supported_card_type_icons": [
-      "american_express",
-      "diners",
-      "discover",
-      "mastercard",
-      "visa"
-    ]
-  },
+    "settings": {
+        "hide_breadcrumbs": false,
```

Comment:
@AlbertJBurton it looks like your editor converted spaces to tabs or something like that here - can you fix that to minimize this changeset to just the functional bits?

Reply:
Updated the config.json file with an editor that doesn't reformat the spacing and tabs in the file.